first-30-minutes-of-the-writing-task-in-studieproeven.md

```diff
@@ -12,9 +12,9 @@ In this blog post, you will learn about the rule of the first 30 minutes for the
 
 For the Writing task (let us call it further an essay), you receive an exam set with three structurally similar tasks but given on three different topics. You can choose between the three tasks, depending on what topic seems more comfortable to write about or familiar to you. All three tasks contain a compulsory question in the assignment description:
 
-*‘Punkt 1: Gør ud fra faktainformationen kort rede for …’*
+> Punkt 1: Gør ud fra faktainformationen kort rede for …
 
-So the task is to describe the main points in the data set usually given in a graphical representation (a table, a diagram, a figure or a pie-chart) about a specific process, a social or an economic issue, or a social trend over a period time. This data description is usually the first logical part of the first ‘real’ paragraph of the essay after an optional short introduction (Introduction is not compulsory and is not on the list of the assessment criteria). You can find an example of the previous year’s exam set here: Studieprøven, November-December 2019. https://uim.dk/arbejdsomrader/danskundervisning-og-prover-for-udlaendinge/prover/danskprover/eksempler-pa-tidligere-afholdte-prover
+So the task is to describe the main points in the data set usually given in a graphical representation (a table, a diagram, a figure or a pie-chart) about a specific process, a social or an economic issue, or a social trend over a period time. This data description is usually the first logical part of the first ‘real’ paragraph of the essay after an optional short introduction (Introduction is not compulsory and is not on the list of the assessment criteria). You can find an example of the previous year’s exam set here: [Studieprøven, November-December 2019](https://uim.dk/arbejdsomrader/danskundervisning-og-prover-for-udlaendinge/prover/danskprover/eksempler-pa-tidligere-afholdte-prover).
```

Comment:
Instead of a link like that:

> https://uim.dk/arbejdsomrader/danskundervisning-og-prover-for-udlaendinge/prover/danskprover/eksempler-pa-tidligere-afholdte-prover

I suggest a link like that:

> [Studieprøven, November-December 2019](https://uim.dk/arbejdsomrader/danskundervisning-og-prover-for-udlaendinge/prover/danskprover/eksempler-pa-tidligere-afholdte-prover)